### PR TITLE
Implemented Radio,Checkbox & switch

### PR DESCRIPTION
**Pull request by edson-pro**

### CODE REVIEW

File: site/components/ui/Checkbox.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  onChange: React.ChangeEventHandler<HTMLInputElement>;
+  checked?: boolean;
+}
+
+export default function Checkbox({ label, onChange, checked = false }: props) {
+  return (
+    <div className="flex my-2 items-center">
+      <div>
+        <input
+          className="form-check-input appearance-none h-4 w-4 border border-gray-500 rounded-sm bg-transparent checked:bg-primary checked:border-primary focus:outline-none transition duration-200 my-1 align-top bg-no-repeat bg-center bg-contain float-left mr-2 cursor-pointer"
```
Comment (hirwaaldo1) [inline site/components/ui/Checkbox.tsx:15]: here you can format the codes.

File: site/styles/globals.css [hunk 2]
```diff
@@ -3,4 +3,15 @@
...
 @tailwind utilities;
-/* HNPRO */
+
+body {
+  @apply bg-gray-900;
+}
+
+.form-check-input:checked[type="radio"] {
+  background-image: url("data:image/svg+xml,%3Csvg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%22-4 -4 8 8%22%3E%3Ccircle r=%222%22 fill=%22%23fff%22/%3E%3C/svg%3E");
+}
```
Comment (hirwaaldo1) [inline site/styles/globals.css:11]: please configure this URL in the tailwind config file.

**Review by hirwaaldo1 — CHANGES_REQUESTED** [review]
things look good but if you can change the following it would be great.

File: site/components/ui/Checkbox.tsx (hunk 1 above)
Comment (edson-pro) [reply site/components/ui/Checkbox.tsx:15]: it's the github view

File: site/styles/globals.css (hunk 2 above)
Comment (edson-pro) [reply site/styles/globals.css:11]: its a data image so it can be hard to do it in tailwind

**Review by hirwaaldo1 — APPROVED** [review]

**Pull request merged**